### DISCUSSION TITLE
Drag and drop tasks

### DISCUSSION THREAD

**Discussion by citiz3n404** [ransome1/sleek#469] [Ideas]
It would be great If we could drag and drop tasks from a priority to another, tasks/substask, or just reorder the order of the tasks.

**Comment by mfvianna** [comment]
In addition th drag tasks to a diferent order, a different priority and todo.txt file, each task could also have small up/down arrows in the task line so that once clicked it would move the task up/down to the line above/below

**Comment by cldjr1** [comment]
I agree!
